tools/tests/test_data_builder.py

```diff
@@ -27,11 +27,16 @@ def test_handle_missing_name(capsys):
     assert "Failed name parse: missing name for" in str(captured) 
 
 
-@pytest.mark.skip(reason="failing test, need to handle this case")
-def test_handle_name_with_multiple_pipes():
+def test_handle_name_with_multiple_pipes(capsys):
     malformed_title = 'Thing happened | this day | May 30th'
-    result = title_to_name_date(malformed_title)
-    # what should happen here?
+    title_to_name_date(malformed_title)
+
+    captured = capsys.readouterr()
```

Comment:
Personally, I don't love this kind of test. I find it's very flaky. If we change the error message, then we have to change the test.

My preference would be to do one of the following:
a) check the output
b) if you still use capsys, then maybe write an error function helper that you call instead
c) use mocks to check like an assert_called or something